src/GraphQL.Tests/Bugs/Bug138DecimalPrecisionTests.cs

```diff
@@ -4,22 +4,25 @@ namespace GraphQL.Tests.Bugs;
 
 public class Bug138DecimalPrecisionTests : QueryTestBase<DecimalSchema>
 {
-#if NETCOREAPP3_1
     [Fact]
-#else
-    [Fact(Skip = "24.149999999999999 with .NET Core < 3.1")]
-#endif
-    public void double_to_decimal_does_not_lose_precision()
+    public async Task double_to_decimal_does_not_lose_precision()
     {
         var query = """
                 mutation SetState{
                     set(request:24.15)
                 }
             """;
 
-        var expected = """{ "set": 24.15 }""";
-
-        AssertQuerySuccess(query, expected);
+        var result = await new DocumentExecuter().ExecuteAsync(new()
```

Comment:
#3473 allows to preserve all internal settings in executer and serializers and do not duplicate it here.